spec/dummy/config/database.yml

```diff
@@ -10,6 +10,8 @@ postgresql:
   test: &pg_test
     <<: *default
     adapter: 'postgresql'
+    username: <%= ENV['POSTGRES_USER'] %>
+    host: localhost
```

Comment:
CI failed on me god knows how many times because I couldn't figure out how to configure this. This worked, but I think any combination of username/password or username `POSTGRES_HOST_AUTH_METHOD=true` would as long as we set `host` here. LMK if there's a better way to do this (since I mostly went with existing configs in other gems)